modules/datafusion/README.md

```diff
@@ -0,0 +1,45 @@
+# Google Cloud DNS Module
+
+This module allows simple management of ['Google Data Fusion'](https://cloud.google.com/data-fusion) instances. It supports creating Basi or Enterprise, public or private instances. 
+
+## Example
+
+```hcl
+module "datafusion" {
+  source     = "./modules/datafusion"
+  name       = "my-datafusion"
+  region     = "europe-west1"
+  project_id = "my-project"
+  network    = "my-network-name"
+}
+```
+
```

Comment:
Can you also add an example that shows how to set the `shared_network_config` variable?

Reply:
Done